src/main/java/de/retest/web/WebData.java

```diff
@@ -93,4 +93,21 @@ public int getAsInt( final String key ) {
 		throw new ConversionException( "Don't know how to convert " + value + " of "
 				+ (value != null ? value.getClass() : "null") + " to int!" );
 	}
+
+	public boolean isShown() {
+		final Boolean shown = (Boolean) wrappedData.get( "shown" );
+		if ( shown != null && !shown ) {
+			return false;
+		}
+		final Rectangle outline = getOutline();
+		if ( outline == null
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this if-then-else statement by a single return statement. ([squid:S1126](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1126))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=94&issues=AWbtpGavsSodgw_KFaHd&open=AWbtpGavsSodgw_KFaHd)

[ISSUE_KEY]: AWbtpGavsSodgw_KFaHd